# Exceptions

Exceptions in Python are events that disrupt the normal flow of a program's execution. They occur when the interpreter encounters an error during runtime, such as trying to divide by zero or accessing an index that's out of bounds in a list. When an exception occurs, Python creates an exception object. If the exception isn't handled, the program will terminate and display an error message. However, you can use `try` and `except` blocks to catch and handle exceptions, allowing your program to continue running even when errors occur.

Visit the following resources to learn more:

- [@official@Errors and Exceptions](https://docs.python.org/3/tutorial/errors.html)
- [@article@Python Exceptions: An Introduction](https://realpython.com/python-exceptions/)
- [@video@Learn Python EXCEPTION HANDLING in 5 minutes!](https://www.youtube.com/watch?v=V_NXT2-QIlE)